packages/yourloops/pages/notifications/notification.tsx

```diff
@@ -257,7 +251,7 @@ export const Notification = (props: NotificationProps): JSX.Element => {
   }
 
   const onDecline = async (): Promise<void> => {
-    setInProgress(true)
+    // setInProgress(true)
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🔨 issue:** ‏Commented code should be removed. This gives the impression that the PR is not ready for review yet, is it the case?